<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="AKASH" failures="2" tests="2" name="Test(failed)" time="0.072" errors="0" timestamp="2021-03-31T20:27:29 IST">
  <testcase classname="failedTestNG.Sample1" name="m3" time="0.013">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failedTestNG.Sample1.m3(Sample1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m3 -->
  <testcase classname="failedTestNG.Sample3" name="m6" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failedTestNG.Sample3.m6(Sample3.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m6 -->
  <testcase classname="failedTestNG.Sample1" name="m1" time="0.0">
    <ignored/>
  </testcase> <!-- m1 -->
  <testcase classname="failedTestNG.Sample1" name="m2" time="0.0">
    <ignored/>
  </testcase> <!-- m2 -->
  <testcase classname="failedTestNG.Sample3" name="m7" time="0.0">
    <ignored/>
  </testcase> <!-- m7 -->
</testsuite> <!-- Test(failed) -->
